src/utils/geo.js

```diff
@@ -200,11 +200,33 @@ export function isProperGeo(countryList = [], name = undefined) {
 	);
 }
 
-export default {
+/**
+ * Select Instant Globals to send them to DFP (aka Ad Manager).
+ *
+ * @param {string[] | undefined} wfKeyVals mapping of Instant Globals (to send) to value in DFP
+ */
+export function getDfpLabradorKeyvals(wfKeyVals) {
+	if (!wfKeyVals || !wfKeyVals.length) {
+		return '';
+	}
+
+	const labradorVariables = module.getSamplingResults();
+
+	return wfKeyVals
+		.map(keyVal => keyVal.split(':'))
+		.filter(keyVal => labradorVariables.indexOf(keyVal[0]) !== -1)
+		.map(keyVal => keyVal[1])
+		.join(',');
```

Comment:
We don't need to join it right? It's fine to set array on targeting. GPT is going to handle that.

Reply:
I did not know that. If we send an array then shouldn't we return an empty array instead of empty string by default?

Reply:
Yes, it will be fine too

Reply:
Returning and empty array will require changes in all other repos - in all of them there is a check `if (keyVals)`, which will evaluate to `true` even for empty arrays :D